playbook-website/app/javascript/components/KitSearch.tsx

```diff
@@ -5,22 +5,22 @@ import { Typeahead } from 'playbook-ui'
 
 type KitSearchProps = {
   classname: String,
-  kits: Array,
```

Comment:
This is not the right way to specify and Array in TS. Since the file needed to be `.tsx`, this had to change in order to transpile.